hopp/simulation/technologies/pv/pv_plant.py

```diff
@@ -73,6 +78,30 @@ class PVPlant(PowerSource):
     def __attrs_post_init__(self):
         system_model = Pvwatts.default(self.config_name)
 
+        
+        if isinstance(self.config.panel_tilt_angle,str):
+            if self.config.panel_tilt_angle == "lat":
+                tilt = self.site.lat
+            elif self.config.panel_tilt_angle == "lat-func":
+                if self.site.lat<=25:
+                    tilt = self.site.lat*0.87
+                elif self.site.lat>25 and self.site.lat<=50:
+                    tilt = (self.site.lat*0.76) + 3.1
+                else:
+                    tilt = self.site.lat
+            else:
+                raise Exception("The str can be set to 'lat' or 'lat-func'.")
+                
+        elif isinstance(self.config.panel_tilt_angle,float):
+            if self.config.panel_tilt_angle > 0 and self.config.panel_tilt_angle<90:
```

Comment:
Is there a reason to exclude 0 and 90?

Reply:
Not it doesn't appear that PySAM is exclusive so I've updated it to include 0 and 90.